google-cloud-text_to_speech/Rakefile

```diff
@@ -0,0 +1,145 @@
+# Copyright 2018 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "bundler/setup"
+require "bundler/gem_tasks"
+
+require "rubocop/rake_task"
+RuboCop::RakeTask.new
+
+desc "Run tests."
+task :test do
+  $LOAD_PATH.unshift "lib", "test"
+  Dir.glob("test/**/*test.rb")
+    .reject { |file| file.include? "smoke_test" }
+    .each { |file| require_relative file }
+end
+
+namespace :test do
+  desc "Runs tests with coverage."
+  task :coverage do
+    require "simplecov"
+    SimpleCov.start do
+      command_name "google-cloud-text_to_speech"
+      track_files "lib/**/*.rb"
+      add_filter "test/"
+    end
+
+    Rake::Task["test"].invoke
+  end
+end
+
+# Acceptance tests
+desc "Run the google-cloud-text_to_speech acceptance tests."
+task :acceptance, :project, :keyfile do |t, args|
+
+  Rake::Task["acceptance:run"].invoke
+end
+
+namespace :acceptance do
+  task :run do
+    puts "The google-cloud-text_to_speech gem does not have acceptance tests."
+  end
+
+  desc "Run acceptance tests with coverage."
+  task :coverage do
+  end
+
+  desc "Run acceptance cleanup."
+  task :cleanup do
+  end
+end
+
+require "yard"
+require "yard/rake/yardoc_task"
+YARD::Rake::YardocTask.new
+
+desc "Generates JSON output from google-cloud-text_to_speech .yardoc"
+task :jsondoc => :yard do
+  require "rubygems"
+  require "gcloud/jsondoc"
+
+  registry = YARD::Registry.load! ".yardoc"
+
+  toc_config = {
+    documents: [
+      {
+        type: "toc",
+        title: "Google::Cloud::Texttospeech::V1::DataTypes",
+        modules: [
+          {
+            title: "Google::Cloud::Texttospeech::V1",
+            include: ["google/cloud/text_to_speech/v1"]
```

Comment:
Is this path incorrect? I think it should be `google/cloud/texttospeech/v1` rather than `google/cloud/text_to_speech/v1`, although I'm not 100% certain.

Reply:
It would be nice to have the upcoming protobuf change here, so we can use `Google::Cloud::TextToSpeech` everywhere.

Reply:
Fixed the path, and removed the protobuf and rpc paths since they aren't actually being generated.

As for the naming change, that's beyond my current knowledge of what's going on. I'll try to find out, but, given that the current generator is using the current naming convention, and that it seems to be a very high priority to get this library out, I think we should keep it as-is for now.

Reply:
I don't see a [3.6.0 gem release yet](https://rubygems.org/gems/google-protobuf/versions). How long does it usually take between library release and gem push?

> given that the current generator is using the current naming convention, and that it seems to be a very high priority to get this library out, I think we should keep it as-is for now.

Yeah, I didn't mean to delay the merge or release, just that the problem of the two different namespaces (`Google::Cloud::TextToSpeech` and `Google::Cloud::Texttospeech`) is going to be resolved soon.

Reply:
I'm not sure how long it takes. The release was tagged over a week ago, and when I asked about it I was told to expect all the artifacts to follow "soon." I'll let you know if I find out more.